admin/system/news-save.php

```diff
@@ -0,0 +1,26 @@
+<?php
+    include_once("connection.php");
```

Comment:
Codacy found an issue: ["include_once" statement detected. File manipulations are discouraged. Statement is not a function, no parentheses are required.](https://app.codacy.com/manual/anamnafiul99/pindad-clone/pullRequest?prid=5762293)